src/fsharp/ErrorLogger.fs

```diff
@@ -273,7 +273,7 @@ module ErrorLoggerExtensions =
     open System.Reflection
 
     // Instruct the exception not to reset itself when thrown again.
-    // Why don?t we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
+    // Why don't we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
     // To achieve this, we need to catch the exception, report the email address and stack trace, and then reraise. 
```

Comment:
by catching the exception, prompting and then propagating the exception with reraise.